decidim-core/app/cells/decidim/attachments_link_tab_cell.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Decidim
+  class AttachmentsLinkTabCell < Decidim::ViewModel
+    alias form model
+
+    def show
+      render :show
+    end
```

Comment:
I think this method is not required, the default behavior is to render the show template

Reply:
oh, I saw it in other cells and I thought it was nice to have it, even if it's not needed it helps following the code, specially for people that are not familiar with `cells`.